### PR TITLE
4 Card components

### PR DESCRIPTION
**Pull request by KaustubhM1997**

### CODE REVIEW

**Comment by KaustubhM1997** [comment]
image
image

File: Cards/cards2.css [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+}
+
+/* .section-ow {
+
+    width: 10rem;
+    /* position: absolute;
+    top: 8rem;
+    left: 15rem; */
+   
+/* } */
```
Comment (Varu98) [inline Cards/cards2.css]: these might not be needed

File: Cards/cards2.css [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+    border: 2px;
+    /* background-color: white; */
+    display: flex;
+    flex-direction: column;
+    /* width: 38.5%; */
+    /* display: flex;
+    flex-direction: column;
+    justify-content: center; */
+    /* margin-left: 15rem; */
+    border-radius: 0.5rem;
```
Comment (Varu98) [inline Cards/cards2.css]: can we change these lines also ?

**Review by Varu98 — APPROVED** [review]
maybe some minor refactoring can be done later on, other-wise everything looks great 👍🏻

**Review by KoushikKintali — APPROVED** [review]
-  Remove the commented code
- folder & file names should be in small letters (best practice)
- rename filename as cards.css instead of cards2.css

File: Cards/cards2.css (hunk 1 above)
Comment (KaustubhM1997) [reply Cards/cards2.css]: Thank you, they have been removed

**Pull request merged**